util/dashboard/gen_dashboard_entry.py

```diff
@@ -45,6 +45,9 @@ def genout(outfile, msg):
     'S1': 'Functional',
     'S2': 'Complete',
     'S3': 'Stable',
+    # In case certain design stages do not apply (e.g. verification
```

Comment:
Nit: Maybe drop "design"? (Since the comments above use "Design Stages" only for D1..D3)

Reply:
yeah that should be `development stages` instead - fixed!